app/frontend/src/components/Basics/_layout.scss

```diff
@@ -69,6 +69,18 @@ $columns: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12;
       }
     }
   }
+
+  @each $name in map.keys($gap-abbrs) {
+    @each $size in $spacing-sizes {
+      .#{$name}-#{$size} > & {
+        // recalculates column size to accommodate gaps
+        @if list.index("gap" "gap-x", $name) {
+          // flex: 0 1 calc(8.33% * #{$col} -  (#{$size} * 8px) * (1 - #{$col} / 12));
+          flex: 0 1 calc(8.33% * $col - ($size * 8px) * (1 - $col / 12));
```

Comment:
Hmm, this works, but it does tend to take away more space from smaller columns than bigger ones, leading to big columns appearing bigger and smaller columns smaller.

It's not a big deal, I think, since I suspect we'll use gaps only in very rare cases, and it might be a big lift to remove width via column ratios.

Reply:
Yes, I did this math assuming all columns have the same width. The tricky thing about trimming widths proportionately is that I don't know how to get the number of columns/gaps in the flexbox in Sass/CSS. I feel like it might involve JS, but I need to research how to run a script and import values to this .scss file or generate SASS in a JS file. This may take a while, so I'll merge this PR now.

Alternatively, we could use `grid `instead of `flexbox` with gaps. Grid will refer to its `grid-template-column` and maintain the ratio of columns when applying gaps in between. Example: https://codepen.io/bzzz-coding/pen/WNawreW

Reply:
Sounds good.